src/libs/l2ps/zk/L2PSBatchProver.ts

```
@@ -0,0 +1,584 @@
+     */
+    private subprocessRequest<T>(type: string, data?: any): Promise<T> {
+        return new Promise((resolve, reject) => {
+            if (!this.childProcess || !this.processReady) {
+                reject(new Error('Subprocess not available'));
+                return;
+            }
+
+            const id = `req_${++this.requestCounter}`;
+            const request = JSON.stringify({ type, id, data }) + '\n';
+
+            this.pendingRequests.set(id, { resolve, reject });
```

ESLint

[error] Extra semicolon.

(semi)